xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/DeletePageConfirmationPage.java

```diff
@@ -69,37 +69,42 @@ public boolean isRecycleBinOptionsDisplayed()
     }
 
     /**
-     * @return the arrow element used for opening or closing the backlinks panel
-     * @since 14.4RC1
+     * Toggle the backlinks panel
+     *
+     * @since 14.4.1
+     * @since 14.5RC1
      */
-    public WebElement getBacklinkPanelExpandButton()
+    public void toggleBacklinksPanel()
```

Comment:
"panel" is confusing to me. I know the CSS class contains 'panel' but at the UI level, for the end user, it's not a panel (on the left or right of the page). We usually ca it **pane**. So ``toggleBacklinksPane``.